BedrockServer.h

```diff
@@ -497,4 +497,6 @@ class BedrockServer : public SQLiteServer {
     // syncNode while the sync thread exists, it's a shared pointer to allow for the last socket thread using it to
     // destroy the pool at shutdown.
     shared_ptr<SQLitePool> _dbPool;
+
+    void notifyPlugins(SQLiteNodeState newState);
```

Comment:
```suggestion
    void notifyPlugins(SQLiteNodeState newState) override;
```

Making sure this won't compile if it's not overriding the original function